client/script.js

```diff
@@ -1,30 +1,41 @@
-
 button = document.querySelector('#searchButton');
 searchedTerm = document.querySelector('#search-bar');
+LuckyButton = document.querySelector('#LuckyButton')
 const ul = document.querySelector('ul');
 
 
 
-
 button.addEventListener('click', function SearchResults(event) {
     let searchTerm = searchedTerm.value.toLowerCase();
-    fetch('http://localhost:3000/search?description=' + searchTerm).then(function (response) {
-        response.json().then(function (json) {
-        for(item in json){
-            for(let details = 0; details < json[item].length; details++){
-                let list = document.createElement("li")
-                let text = document.createTextNode(`${json[item][details].Name},${json[item][details].description},${json[item][details].url}`);
-                list.appendChild(text);
-                 ul.appendChild(list);
-                
-
-
-
+    if(searchTerm === ""){
+        alert("search input empty");
+    }
+    else{
+        window.location = 'http://localhost:3000/search?description=' + searchTerm;
```

Comment:
This response displays the API as a JSON, is this making your for functions not do anything? You've done well to fetch the information from the API but if you can get that to directly edit the user HTML then the results would display like google and not just show the API. Perhaps you've thought this but couldn't get it to work in practice.